### ISSUE TITLE
New lines in text fields are doubled

### ISSUE THREAD

**Issue by svivian** [phpmyadmin/phpmyadmin#16765]
### To Reproduce

1. Browse a table that has a TEXT field, with multiple lines
2. Click to edit inline, make some changes
3. Press Enter or click away to save

### Expected behavior

The text to be saved as written.

### Screenshots

Every line break is duplicated, so if you have e.g.

    My text
    here

It saves as

    My text

    here

### Server configuration

Same issue on localhost and the PMA demo server.

- Operating system: Mac/Linux
- Web server: Apache
- Database version: 5.7.24 / 8.0.18
- PHP version: 7.4
- phpMyAdmin version: 5.1.0 (via git STABLE branch)

### Client configuration

- Browser: Chrome
- Operating system: MacOS 11.2.3

**Comment by williamdes** [comment]
Hi @svivian 
Thank you for reporting this issue, I could reproduce it. Indeed there is some transformation done.
I tried to quick search for the cause but will have to continue searching :)

**Comment by williamdes** [comment]
I tracked down the bug and found #12918 (55880451eb193f7e1bad90df365b67d54eded5a6).
I think this was recently introduced by #16095 (2c7b7046e206362ddb8fe5c3b41ede91d43361da)
And that before that the display was good
Disabling css class `.pre_wrap` seems to remove the issue.

Maybe #12918 should be reverted now

cc @petrduda

**Comment by svivian** [comment]
Just an FYI, I reverted to 5.0.5 (branch `QA_5_0`) and the second part of the issue still occurs there - in other words when you click out of the textarea the correct text is shown, but if you click to edit the field again, it _does_ have the duplicate line breaks.

**Comment by williamdes** [comment]
> Just an FYI, I reverted to 5.0.5 (branch `QA_5_0`) and the second part of the issue still occurs there - in other words when you click out of the textarea the correct text is shown, but if you click to edit the field again, it _does_ have the duplicate line breaks.

Could you checkout QA_5_1 and remove 55880451eb193f7e1bad90df365b67d54eded5a6 from the js dist or js src and re-build the JS files ?
Could you confirm that removing 55880451eb193f7e1bad90df365b67d54eded5a6 does fix all issues?

**Comment by svivian** [comment]
Yes, that does appear to fix both the table display and the textarea when re-edited.
